src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -274,14 +273,57 @@ def verify_usr_attr(self, cont):
                       len(attrs.keys()), attrs.items())
 
     @avocado.fail_on(DaosApiError)
-    def test_dm_dst_create_dcp(self):
+    def test_dm_dst_create_dcp_posix_dfs(self):
+        """
+        Test Description:
+            Verifies destination container creation
+            for DFS API, including
+            container properties and user attributes.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=datamover,dcp
+        :avocado: tags=dm_dst_create,dm_dst_create_dcp_posix_dfs
+        """
+        self.run_dm_dst_create("DCP", "POSIX", "DFS", True)
+
+    @avocado.fail_on(DaosApiError)
+    def test_dm_dst_create_dcp_posix_daos(self):
         """
         Test Description:
-            Verifies destination container creation, including
+            Verifies destination container creation
+            for POSIX containers using OBJ API, including
             container properties and user attributes.
 
         :avocado: tags=all,full_regression
         :avocado: tags=datamover,dcp
-        :avocado: tags=dm_dst_create,dm_dst_create_dcp
+        :avocado: tags=dm_dst_create,dm_dst_create_dcp_posix_daos
+        """
+        self.run_dm_dst_create("DCP", "POSIX", "DAOS", True)
+
+    @avocado.fail_on(DaosApiError)
+    def test_dm_dst_create_dcp_unknown_daos(self):
+        """
+        Test Description:
+            Verifies destination container creation
+            when API is unknown, including
+            container properties and user attributes.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=datamover,dcp
+        :avocado: tags=dm_dst_create,dm_dst_create_dcp_unknown_daos
+        """
+        self.run_dm_dst_create("DCP", None, "DAOS", True)
+
+    @avocado.fail_on(DaosApiError)
+    def test_dm_dst_create_fs_copy_posix_dfs(self):
+        """
+        Test Description:
+            Verifies destination container creation
+            when API is unknown, including
+            container properties and user attributes.
```

Comment:
It looks like this is the same description as the method above.

Reply:
> It looks like this is the same description as the method above.

@shimizukko thanks, since these are for documentation/descriptions I'd like to update them on my next PR to the filesystem copy tool. It took days for this PR to get through CI and it is fixing an important bug as well.